ccm_web/server/appHandler.ts

```diff
@@ -7,7 +7,8 @@ import Database from 'ltijs-sequelize'
 import { Config } from './config'
 
 interface DevOptions {
-  isDev: true
+  isDev: true,
+  staticPath: string
 }
```

Comment:
I think if we make these changes, we can just coalesce `ProdOptions` and `DevOptions` into an `EnvOptions` interface that looks like this

```
interface EnvOptions {
  isDev: boolean
  staticPath: string
}
```

I was doing it the other way before because `staticPath` was conditionally available.

Reply:
Done